op-node/rollup/derive/deposit_log_test.go

```diff
@@ -47,8 +50,10 @@ type receiptData struct {
 	DepositLogs []bool
 }
 
-func makeReceipts(rng *rand.Rand, blockHash common.Hash, depositContractAddr common.Address, testReceipts []receiptData) (receipts []*types.Receipt, expectedDeposits []*types.DepositTx) {
+func makeReceipts(rng *rand.Rand, blockHash common.Hash, depositContractAddr common.Address, testReceipts []receiptData) ([]*types.Receipt, []*types.DepositTx, error) {
 	logIndex := uint(0)
+	receipts := []*types.Receipt{}
+	expectedDeposits := []*types.DepositTx{}
```

Comment:
I realize this is kind of ugly, but the alternative would've been to name the error variable in the function declaration, which seemed like it went against convention in the rest of the codebase.

Reply:
we've done it, but I actually prefer explicit rather than named returns

Reply:
Yeah, this instance of named returns seemed unconventional as well.